src/features/callAssignments/rpc/fixNewCallAssignmentFilterConfig.ts

```diff
@@ -0,0 +1,49 @@
+import { z } from 'zod';
+
+import IApiClient from 'core/api/client/IApiClient';
+import { makeRPCDef } from 'core/rpc/types';
+import { ZetkinQuery } from 'utils/types/zetkin';
+import {
+  CALL_OPERATOR,
+  FILTER_TYPE,
+} from 'features/smartSearch/components/types';
+
+const paramsSchema = z.object({
+  callAssignmentId: z.number(),
+  orgId: z.number(),
+  queryId: z.number(),
+});
+
+type Params = z.input<typeof paramsSchema>;
+type Result = ZetkinQuery;
+
+export const fixNewCallAssignmentFilterConfigDef = {
+  handler: handle,
+  name: 'updateQuery',
```

Comment:
This should reflect the name in the code, in this case `fixNewCallAssignmentFilterConfigDef`, although I think that name could be simplified.